docs/In-Band Manageability User Guide - ThingsBoard.md

```diff
@@ -1180,8 +1180,35 @@ dashboard to trigger a Device Reboot.
 Now on the pop-up window shows up, click the `Send` button on the box
 titled **`Decommission Device`**.
 
-### Query Commands
+### Query Command
 
+The Intel® In-Band Manageability provides a way to query attribute information on either the Host, Edge Device, or Nodes.
+
+To trigger a **Trigger Query**, follow the steps below:
+
+In order to query updates, click the **Trigger Query button** as seen below.
+
+<img src="media/In-Band Manageability User Guide - ThingsBoard/media/image65.png" style="width:5.78125in;height:3.59762in" />
+
+Populate the **Trigger Query** pop-up window with the required parameters and click **send** to trigger the query Update.
+
+<img src="media/In-Band Manageability User Guide - ThingsBoard/media/image64.png" style="width:5.78125in;height:3.59762in" />
+
+**Find the Query Option and the Description below:**
+
+
```

Comment:
As with the Azure document, the Query options should be referenced from the common document instead of duplicated.